mm2src/lp_swap/taker_swap.rs

```diff
@@ -1024,9 +1025,18 @@ impl TakerSwap {
         let transaction = match fee_tx {
             Ok(t) => t,
             Err(err) => {
+                if let Some(tx) = err.get_tx() {
```

Comment:
It seems that during review sessions, I already forgot the initial requirement :slightly_smiling_face: 
As per issue https://github.com/KomodoPlatform/atomicDEX-API/issues/1238, this should be done only in:
`spend_taker_payment`
`refund_maker_payment`
`spend_maker_payment`
`refund_taker_payment`

We need `broadcast_p2p_tx_msg` only in cases, when it's essential to spend other side's payment or refund ours.
In other cases, `broadcast_p2p_tx_msg` may even be harmful. E.g., in `send_taker_fee` our node fails to broadcast the dex fee initially, but manages to broadcast it through P2P helper. As it finishes the swap, the taker will lose the fee.